evap/results/exporters.py

```diff
@@ -153,65 +129,55 @@ def export(self, response, course_types_list, include_not_enough_voters=False, i
 
                     for course, results in courses_with_results:
                         if questionnaire.id not in results or question.is_heading_question:
-                            self.write_two_empty_cells_with_borders()
+                            self.write_empty_cell_with_borders()
                             continue
                         qn_results = results[questionnaire.id]
                         values = []
-                        deviations = []
                         total_count = 0
                         approval_count = 0
 
                         for grade_result in qn_results:
                             if grade_result.question.id == question.id:
                                 if grade_result.average:
                                     values.append(grade_result.average * grade_result.total_count)
-                                    deviations.append(grade_result.deviation * grade_result.total_count)
                                     total_count += grade_result.total_count
                                 if grade_result.question.is_yes_no_question:
                                     approval_count += grade_result.approval_count
```

Comment:
this throws `unsupported operand type(s) for +=: 'int' and 'NoneType'` when i tested exporting